scripts/config/msu/string.nut

```diff
@@ -27,4 +27,14 @@
 		}
 		return true;
 	}
+
+	function startsWith( _string, _start )
+	{
+		return _string.find(_start) == 0;
+	}
+
+	function endsWith( _string, _end )
+	{
+		return _end.len() <= _string.len() && _string.slice(-_end.len()) == _end;
+	}
```

Comment:
Let me be the stickler this time and say that this belongs in a different commit

Reply:
That's fair, not sure how this happened.